lib/util/memcache.ex

```diff
@@ -6,10 +6,50 @@ defmodule Antikythera.Memcache do
   @moduledoc """
   In-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  Antikythera provides in-memory cache controlled by AntikytheraCore for each executor pool.
+  Objects are stored in ETS internally.
```

Comment:
We should stick with the same terminology for the cached values.
In most part "record" is used; here it's "object".

Combining @ymtszw's comment above, try this:
```
Easy-to-use in-memory cache for each executor pool.

#{inspect(__MODULE__)} behaves as a key-value storage.
Cached key-value pairs are internally stored in ETS.
It accepts arbitrary (but not too large) terms as both keys and values.
```

Reply:
FIxed in e8adcb50f657f23724421e7b0383d11d95fe9bb0